server/src/main/java/org/apache/iotdb/db/engine/merge/task/MergeFileTask.java

```diff
@@ -19,11 +19,17 @@
 
 package org.apache.iotdb.db.engine.merge.task;
 
+import static org.apache.iotdb.tsfile.common.constant.TsFileConstant.TSFILE_SUFFIX;
+
 import java.io.File;
 import java.io.IOException;
+import java.util.HashMap;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.Set;
```

Comment:
unused import can be deleted by idea using ```control+option+O```

Reply:
okay, I will fix it